src/github/push.ts

```diff
@@ -28,6 +34,18 @@ export const pushWebhookHandler = async (context: WebhookContext, jiraClient, _u
 		gitHubInstallationId
 	});
 
+	if (await booleanFlag(BooleanFlags.CONFIG_AS_CODE, false, jiraClient.baseURL)) {
+		if (subscription) {
+			const repoSyncState = await RepoSyncState.findByRepoId(subscription, payload.repository.id);
+			const modifiedFiles = context.payload?.commits?.reduce((acc, commit) =>
+				([...acc, ...commit.added, ...commit.modified, ...commit.removed]), []);
+			// TODO: this call must be updated to support GitHub Server events
+			await updateRepoConfig(repoSyncState, getInstallationId(gitHubInstallationId), modifiedFiles);
```

Comment:
Flagging that this call must be updated to support GitHub Server. How do we get the app ID and GitHub base url in the webhook path @joshkay10 @gxueatlassian?

Reply:
there should be `/git/:uuid/webhook/blah` something like that. And in the `GithubServerAppMiddleware`, it will extract the uuid and put it in side `locals` for use. Me or @Harminder84 is working on it atm.